src/app/components/import/AttachingModalContent.js

```diff
@@ -0,0 +1,48 @@
+import React, { useRef } from 'react';
+import PropTypes from 'prop-types';
+import { c } from 'ttag';
+import { Bordered, FileInput, Alert } from 'react-components';
+
+import AttachedFile from './AttachedFile';
+
+const AttachingModalContent = ({ attached, file, onAttach, onClear }) => {
+    const myRef = useRef();
```

Comment:
Unused?

Reply:
yeah, I haven't fully cleaned up the repo yet (still WIP). Sorry!